asciimatics/screen.py

```diff
@@ -1914,6 +1914,7 @@ def close(self, restore=True):
                 curses.echo()
                 curses.nocbreak()
                 curses.endwin()
+                signal.signal(signal.SIGWINCH, signal.SIG_DFL)
```

Comment:
I tried to do that initially, however since you set the signal each time in the `_CursesScreen` initializer, I would need to restore the signals on every close, not just the `restore`=`True` close. If that's fine, then I'll make the change.